client/src/components/DatasetProjectCard.js

```diff
@@ -0,0 +1,144 @@
+// Currently mock data is used via Storybook for development
+export const DatasetProjectCard = ({ dataset, projectId }) => {
+  const { responsive } = useResponsive()
+
+  const {
+    data,
+    stats: { projects }
+  } = dataset
+
+  const projectData = data[projectId]
+  const { merge_single_cell: mergedSingleCell, includes_bulk: includesBulk } =
+    projectData
+
+  const projectStats = projects[projectId]
+  const downloadableSamples = projectStats.downloadable_sample_count
+  const samplesDifferenceCount = projectStats.samples_difference_count
+  const isSamplesDifference = samplesDifferenceCount > 0
+
+  const modalities = ['SINGLE_CELL', 'SPATIAL']
+  const options = ['merge_single_cell', 'includes_bulk']
+  const hasNoOptions = options.filter((o) => projectData[o]).length === 0
+
+  const downloadOptions = [
+    {
+      title: 'Data Format',
+      value: getReadable(dataset.format)
+    },
+    {
+      title: 'Modality',
+      value: modalities.map((modality) => {
+        const totalSamples = projectData[modality].length
+        return (
+          <Box key={modality}>
+            <Text>
+              {totalSamples > 0 && `${getReadable(modality)} (${totalSamples})`}
+            </Text>
+          </Box>
+        )
+      })
+    },
+    {
+      title: 'Other Options',
+      value: (
+        <Box key={projectId}>
+          {includesBulk && (
+            <Text>Include all bulk RNA-seq data in the project</Text>
+          )}
+          {mergedSingleCell && (
+            <Text>Merge single-cell samples into 1 object</Text>
+          )}
+          {hasNoOptions && <Text>Not Specified</Text>}
+        </Box>
+      )
+    }
+  ]
+
+  return (
+    <Box elevation="medium" pad="24px" width="full">
+      <Box
+        border={{ side: 'bottom' }}
+        margin={{ bottom: '24px' }}
+        pad={{ bottom: '24px' }}
+      >
+        <DatasetHeader linked title={projectStats.title} />
+      </Box>
+      <Box margin={{ bottom: '24px' }}>
+        <Badge badge="Samples">
+          <Text size="21px" weight="bold">
+            {downloadableSamples} Sample{downloadableSamples > 1 ? 's' : ''}
+          </Text>
+        </Badge>
+      </Box>
+      <Box>
+        <Box margin={{ bottom: '24px' }}>
+          <Label label="Diagnosis" />
+          <Text>{projectStats.diagnoses_counts}</Text>
+        </Box>
+        <Box margin={{ bottom: 'xsmall' }}>
+          <Label label="Download Options" />
+        </Box>
+        <Box
+          direction={responsive('column', 'row')}
+          margin={{ bottom: '24px' }}
+        >
+          {downloadOptions.map((d) => (
+            <Box key={d.title} flex={{ grow: 1 }}>
+              <Label label={d.title} />
+              {d.value ? (
+                <Text>{d.value}</Text>
+              ) : (
+                <Text italic color="black-tint-30">
+                  Not Specified
+                </Text>
```

Comment:
This shouldn't have to be here since you already have the fall back for "other options"